layouts/index.html

```diff
@@ -35,8 +35,83 @@ <h1>{{.Params.Title}}</h1>
 
 
 
+    
+    <!-- ================================== START OF FEATURED EVENT ============================== -->
+    {{ $eventsData := .Site.Data.events }}
+    {{ $count := 0 }}
+    {{ $showFeaturedEvent := false }}
+    <!-- check if there is any event present or not -->
+    {{ if $eventsData }}
+        {{ $events := sort $eventsData.event "eventOrder" }}
+        <!-- count featured events -->
+        {{ range $event := $events }}
+            {{ if $event.featured }}
+                {{ $count = add $count 1 }}
+            {{ end }}
+        {{ end }}
+
+        <!-- Only display the "Featured event section" if there are more than two events to display. -->
+        {{ if gt $count 2 }}
+            {{ $showFeaturedEvent = true }}
+        {{ end }}
+    {{ end }}
```

Comment:
Currently, we display the featured event if there are at least three events to display; please let me know if this needs to be changed, and I will make the necessary changes.

Reply:
ah, Yes if possible please make it show up even if theres just one

Reply:
ok